tests/fixtures/tests/Passing/TrickyMultilines.elm

```diff
@@ -20,6 +20,8 @@ var $elm_explorations$test$Test$Internal$Batch = function (a) {
   return {$: 'Batch', a: a};
 };
 """
+        -- Remove CR on Windows.
+        |> String.replace "\u{000D}" ""
```

Comment:
What happens with out this line?

Reply:
I would guess this is probably needed when a windows machine checks out this file from git.  If `core.autocrlf` is set to true, then git will insert CRs in the multiline strings.  If that's the cause, then ensuring `core.autocrlf=input` when checking out the project should make this line unnecessary.

Reply:
Yeah, because below we check this string against a manually escaped string. I added a bit of extra explanation which hopefully will prevent my future self from becoming confused again.